RT_SYCL/hitable.hpp

```diff
@@ -19,7 +19,7 @@ class hit_record {
     // To set if the hit point is on the front face
     void set_face_normal(const ray& r, const vec& outward_normal) {
         front_face = dot(r.direction(), outward_normal) < 0;
-        normal = front_face ? outward_normal : -outward_normal;
+        normal = front_face ? outward_normal : vec{}-outward_normal;
```

Comment:
Yes, we forgot unary minus this in the SYCL spec up to recently. I hope it will land in DPC++ too, soon.